nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -21,40 +21,63 @@
-| Region               | `level`                                 | `tree_depth`                                   |
-| Service              | `ipaddresses`                           | `ip_addresses`                                 |
-| Tenant               | `group`                                 | `tenant_group`                                 |
-| TenantGroup          | `level`                                 | `tree_depth`                                   |
-| VirtualMachine       | `local_context_data`                    | `local_config_context_data`                    |
-|                      | `local_context_data_owner_content_type` | `local_config_context_data_owner_content_type` |
-|                      | `local_context_data_owner_object_id`    | `local_config_context_data_owner_object_id`    |
-|                      | `local_context_schema`                  | `local_config_context_schema`                  |
-| VLAN                 | `group`                                 | `vlan_group`                                   |
+| Model                   | Renamed Field                           | New Name                                       |
+|-------------------------|-----------------------------------------|------------------------------------------------|
+| CablePath               | `circuittermination`                    | `circuits_circuittermination_related`          |
+|                         | `consoleport`                           | `dcim_consoleport_related`                     |
+|                         | `consoleserverport`                     | `dcim_consoleserverport_related`               |
+|                         | `interface`                             | `dcim_interface_related`                       |
+|                         | `powerfeed`                             | `dcim_powerfeed_related`                       |
+|                         | `poweroutlet`                           | `dcim_poweroutlet_related`                     |
+|                         | `powerport`                             | `dcim_powerport_related`                       |
+| Circuit                 | `termination_a`                         | `circuit_termination_a`                        |
+|                         | `termination_z`                         | `circuit_termination_z`                        |
+|                         | `terminations`                          | `circuit_terminations`                         |
+|                         | `type`                                  | `circuit_type`                                 |
+| Cluster                 | `group`                                 | `cluster_group`                                |
+|                         | `type`                                  | `cluster_type`                                 |
+| ConfigContextSchema     | `schema`                                | `config_context_schema`                        |
```

Comment:
Might be worthwhile to add a new section to this document ("Git Data Source Changes" or similar) that calls out the need to update git repository data to reflect this change.

We should also consider intentionally implementing a backwards-compatibility shim in the Git datasource code to make it so that both `schema` and `config_context_schema` are accepted and understood for now, perhaps logging a warning if `schema` is still used.